packages/daemon/test/test-endo.js

```diff
@@ -439,6 +439,160 @@ test('store with name', async t => {
   }
 });
 
+test('move renames value', async t => {
+  const { host } = await prepareHost(t);
+
+  await E(host).evaluate('MAIN', '10', [], [], 'ten');
+
+  t.true(await E(host).has('ten'));
+
+  await E(host).move(['ten'], ['zehn']);
+
+  t.false(await E(host).has('ten'));
+  t.true(await E(host).has('zehn'));
+});
+
+test('move renames value, overwriting the "to" name', async t => {
+  const { host } = await prepareHost(t);
+
+  await E(host).evaluate('MAIN', '10', [], [], 'ten');
+  await E(host).evaluate('MAIN', '"X"', [], [], 'decimus');
+
+  t.true(await E(host).has('ten'));
+  t.true(await E(host).has('decimus'));
+
+  await E(host).move(['ten'], ['decimus']);
+
+  t.false(await E(host).has('ten'));
+  t.true(await E(host).has('decimus'));
+
+  const decimusValue = await E(host).lookup('decimus');
+  t.is(decimusValue, 10);
+});
+
+test('move moves value, from the host to a different name hub', async t => {
+  const { host } = await prepareHost(t);
+  const nets = await E(host).lookup('NETS');
```

Comment:
We don’t have `makeDirectory` yet? `NETS` is probably okay, but I wouldn’t count on it remaining that way for non-network shaped things.

Reply:
Thank you for reminding me that `makeDirectory()` exists!